oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -181,18 +183,19 @@ static ServiceAccountCredentials fromJson(
     String clientEmail = (String) json.get("client_email");
     String privateKeyPkcs8 = (String) json.get("private_key");
     String privateKeyId = (String) json.get("private_key_id");
+    String projectId = (String) json.get("project_id");
```

Comment:
Some service account files don't have project_id in them -- will this work for them?

If so, I'll give an LGTM and merge.